### DISCUSSION TITLE
I want to ask How to delete a picture on the webpage?

### DISCUSSION THREAD

**Discussion by UniqueDing** [photoview/photoview#226] [Q&A and Troubleshooting]
I havn't found a delete button,thanks

**Comment by viktorstrate** [comment]
Short answer: You can't, you have to delete it directly on the filesystem and rescan.

However there is a discussion about maybe introducing this, see #216.
The problem is that introducing write support would introduce a lot of complexity and security conserns.

**Reply by UniqueDing** [reply]
I got it! Thanks!!!

**Reply by larsxschneider** [reply]
For me it would be sufficient to mark the images as deleted in the database. Then I can query the database and delete the images in batch via the command line.

**Comment by plrnr** [comment]
This is the only thing that is lacking on photoview. There are just too many pics to delete and going back to shell to delete them is very time consuming. Any possibility deleting pics can be added?

**Comment by Ariczh** [comment]
yep. delete a old photo with a  hash name in shell is a so bad idea.
